src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -1,12 +1,88 @@
 package frc.team2412.robot.subsystem;
 
-import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
-
+import com.ctre.phoenix.motorcontrol.TalonFXControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
 import edu.wpi.first.wpilibj.DoubleSolenoid;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class ClimbSubsystem extends SubsystemBase {
-    public ClimbSubsystem(WPI_TalonFX climbFixed1, WPI_TalonFX climbFixed2, WPI_TalonFX climbAngled1, WPI_TalonFX climbAngled2, DoubleSolenoid climbAngle) {
 
+    public static class ClimbConstants {
+        public static final double MAX_SPEED = 1;
+        public static final double STOP_SPEED = 0;
+    }
+
+    /**
+     * State of the cliimb subsystem
+     */
+    enum ClimbSubsystemState {
+        ENABLED, DISABLED
+    }
+
+    /**
+     * State of the solenoid to change the climb angle
+     */
+    enum HookArmState {
+        EXTENDED, UPRIGHT
+    }
+
+    private final TalonFX climbMotor1;
+    private final TalonFX climbMotor2;
+
+    private final DoubleSolenoid solenoid;
+
+    ClimbSubsystemState state;
+
+    public ClimbSubsystem(TalonFX climbFixed1, TalonFX climbFixed2, DoubleSolenoid climbAngle) {
```

Comment:
Renaming these so they don't imply both are for the fixed arm would also be helpful for readability.